src/components/Jobs/Jobs.test.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../test/jest/__mock__';
+import {
+  jobExecutions,
+  jobsLogs,
+} from '../../../test/bigtest/mocks';
+
+import { translationsProperties } from '../../../test/jest/helpers';
+import { DEFAULT_TIMEOUT_BEFORE_JOB_DELETION } from '../../utils';
+
+import { DataFetcherContext } from '../DataFetcher';
+
+import { Jobs } from './Jobs';
+
+const defaultContext = {
+  hasLoaded: true,
+  jobs: [],
+  logs: jobsLogs,
+};
+
+const renderJobs = (context = defaultContext) => {
+  const component = (
+    <DataFetcherContext.Provider value={context}>
+      <Jobs />
+    </DataFetcherContext.Provider>
+  );
+
+  return renderWithIntl(component, translationsProperties);
+};
+
+const unMockedFetch = global.fetch;
```

Comment:
Can we mock the fetch like for the Accepted values field component?

Reply:
noted.